.ci/releasePipeline.groovy

```diff
@@ -302,6 +327,12 @@ void createContainerAndPushToStorage(Map args = [version: null, tagLatest: false
     }
 }
 
+void pushAndSign(Map args = [imageName: null]) {
```

Comment:
nit: now that it's only used once this might just stay part of the overall createContainer method? It heavily depends on the other parts of that method having built the container